site/content/in-dev/unreleased/configuring-polaris-for-production.md

```diff
@@ -23,114 +23,179 @@ type: docs
 weight: 600
 ---
 
-The default `polaris-server.yml` configuration is intended for development and testing. When deploying Polaris in production, there are several best practices to keep in mind.
+## Tuning Polaris for Production
 
-## Security
+The default server configuration is intended for development and testing. When deploying Polaris in
+production, there are several best practices to keep in mind.
 
-### Configurations
+Notable configuration options used to secure a Polaris deployment are outlined below.
 
-Notable configuration used to secure a Polaris deployment are outlined below.
+For more information on how to configure Polaris and what configuration options are available,
+refer to the Configuration Reference page.
 
-#### oauth2
+### Security
```

Comment:
I feel that this change implies other sections are _not_ related to security, which could be misleading. For example, bootstrap credentials are at least tangentially related to security.